.github/workflows/build-test-publish-images.yml

```diff
@@ -187,6 +187,7 @@ jobs:
       - name: Create multiarch manifest
         shell: bash
         env:
+          RAFT_ANN_BENCH_CPU_IMAGE_BUILT: ${{ matrix.CUDA_TAG == '12.5.1' }}
```

Comment:
Instead of duplicating this `12.5.1` in another place, would you consider using GitHub Actions `outputs`?

I think it should be possible to do something like this example:

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/defining-outputs-for-jobs#using-job-outputs-in-a-matrix-job

Where the `build` job could directly set an output `RAFT_ANN_BENCH_CPU_IMAGE_BUILT` or similar that could then be referenced here.

I think that'd be preferable, because at first glance without the context from the PR description, it's not clear why a specific CUDA version is showing up in this part of the configuration.

Reply:
@jameslamb I've moved the logic to the jq script -- we effectively determine if we should build that image (depending on the CUDA version) right from where we construct the matrix. Thoughts?